### PR TITLE
Internal WiFi block

### PR DESCRIPTION
**Pull request by jacksgt**
New internal block for yabar: "YABAR_WIFI"
Displays (for example) `My Wifi (88%)`

### CODE REVIEW

File: src/intern_blks/ya_intern.c [hunk 1]
```diff
@@ -573,6 +575,103 @@ void ya_int_volume(ya_block_t *blk) {
...
+    }
+    iw_sockets_close(skfd);
+
+    return 0;
+}
+
+void ya_int_wifi(ya_block_t *blk) {
+    struct wireless_stats ws;
+    char *startstr = blk->buf;
+	size_t prflen = 0, suflen = 0;
```
Comment (jacksgt) [inline src/intern_blks/ya_intern.c]: Thanks for the hint! Fixed.

**Comment by jacksgt** [comment]
Huh, `nl80211` is really a great name for a generic library... I'll look into it.
Should we merge the new block anyway (AFAIK it works fine, also we can switch the library calls beneath later on) or not?

**Comment by NBonaparte** [comment]
On a somewhat unrelated note, should we add this to the README or move everything over to the wiki? Otherwise, we can merge it.

**Comment by jacksgt** [comment]
<Off Topic>
> On a somewhat unrelated note, should we add this to the README or move everything over to the wiki?
I was recently thinking about this. I want to have these three features for yabar documentation:
1. easily readable on the web
2. automatic man page generation
3. editable after review

The first requirement is fulfilled by any markdown format (at least on GitHub), however the second one is already a struggle. What I dislike about GitHub wikis is that it can either be edited by **everyone** or only us. I don't like both of those options. Either option is too extreme. Ideally, with every pull request the author should already include the required documentation.

I will look into [asciidoc](http://www.methods.co.nz/asciidoc/), maybe that's an option.

**Pull request merged**